### DISCUSSION TITLE
Guide

### DISCUSSION THREAD

**Discussion by Supersimmons** [janoside/btc-rpc-explorer#540] [Q&A]
HI,
Is there a guide that explains to me the meaning of the entries on the peers page?
Thanks

**Comment by janoside** [comment]
No, there's not. The goal of any particular page is for the data presented to be accompanied by inline explanatory text, often in the form of tooltips via underlined items (admittedly, poor design from an accessibility standpoint :( ). The peers page could likely benefit from some UX reworking, which I'll plan for. Thanks for highlighting this fact.
